openquake/server/manage.py

```diff
@@ -53,6 +53,8 @@ def dbcmd(action, *args):
     if 'runserver' in sys.argv:
         if '--nothreading' in sys.argv:
             logs.dbcmd = dbcmd  # turn this on when debugging
+        # check if the dbserver is the right one
+        dbserver.check_foreign() # check if we are taliking to the right server
         logs.dbcmd('upgrade_db')  # make sure the DB exists
```

Comment:
I would change `check_foreign` to return a string and I would use it as follows:

```python
err = dbserver.check_foreign() 
if err:
    sys.exit(err)
```